### PR TITLE
style: Typos, missing links and broken links

### PR DESCRIPTION
**Pull request by JulienMirval**

### CODE REVIEW

**Comment by paultranvan** [comment]
As you're on it, would you mind changing all `client.find(...` by `Q(...)`? :)

**Comment by JulienMirval** [comment]
> 

Done!

File: src/tutorials/data/advanced.md [hunk 1]
```diff
@@ -40,43 +40,43 @@ This design is very efficient for range queries: in the example above, a query l
...
 ## Indexes: why is my document not being retrieved?
 
-At indexing time, CouchDB evaluates if a new document should be indexed by checking if its fields match an existing index. It means that **all the indexed fields must exist in the document** to index it. It can be problematic for queries that need to check if a particular field exists or not. 
+At indexing time, CouchDB evaluates if a new document should be indexed by checking if its fields match an existing index. It means that **all the indexed fields must exist in the document** to index it. It can be problematic for queries that need to check if a particular field exists or not.
 
 For example, this query won’t work:
 
 ```javascript
 const queryDef = client
   .find("io.cozy.todos")
```
Comment (paultranvan) [inline src/tutorials/data/advanced.md:49]: gotcha!

File: src/tutorials/data/advanced.md [hunk 2]
```diff
@@ -40,43 +40,43 @@ This design is very efficient for range queries: in the example above, a query l
...
 Here, we want to get all the todos without a `category` field. However, those documents will never be indexed, precisely because the field is missing.
 
-Hence, think carefully about your data before hand and do not use queries with `$exists: false` whenever possible. If a field is missing, it might be better to add a  `null` value so that your documents do not have any missing fields. If this is not possible for some reasons, here is another possible solution - yet not very efficient:
+Hence, think carefully about your data before hand and do not use queries with `$exists: false` whenever possible. If a field is missing, it might be better to add a `null` value so that your documents do not have any missing fields. If this is not possible for some reasons, here is another possible solution - yet not very efficient:
 
 ⚠️ Not efficient workaround
 
 ```javascript
 const queryDef = client
   .find("io.cozy.todos")
```
Comment (paultranvan) [inline src/tutorials/data/advanced.md:66]: gotcha!

File: src/tutorials/data/advanced.md [hunk 3]
```diff
@@ -88,13 +88,13 @@ To illustrate this, let’s assume you have declared this query, to get all the
...
```
Comment (paultranvan) [inline src/tutorials/data/advanced.md:89]: gotcha!

File: src/tutorials/data/advanced.md [hunk 4]
```diff
@@ -221,19 +221,19 @@ If you are interested about this, you can learn more [here](https://use-the-inde
...
-In CouchDB, a document is indexed only if its fields matches an existing index. As a consequence, it is not possible to perform efficient queries on non-existing fields. 
+In CouchDB, a document is indexed only if its fields matches an existing index. As a consequence, it is not possible to perform efficient queries on non-existing fields.
 
 For instance, the query below does not work, because it searches for document without the `category` field, but the absence of this field will cause such documents to never be indexed, as previously explained [here](#indexes-why-is-my-document-not-being-retrieved):
 
 ```javascript
 const queryDef = client
   .find("io.cozy.todos")
   .where({
-    "category": {
```
Comment (paultranvan) [inline src/tutorials/data/advanced.md:232]: gotcha!

File: src/tutorials/data/advanced.md [hunk 5]
```diff
@@ -242,18 +242,18 @@ In the same manner, queries implying non-contiguous rows in the B+ Tree are sub-
 const queryDef = client
   .find("io.cozy.todos")
```
Comment (paultranvan) [inline src/tutorials/data/advanced.md:243]: gotcha!

File: src/tutorials/data/advanced.md [hunk 6]
```diff
@@ -263,14 +263,14 @@ You can declare a partial index in cozy-client like this:
...
```
Comment (paultranvan) [inline src/tutorials/data/advanced.md:264]: gotcha!

File: src/tutorials/data/queries.md [hunk 7]
```diff
@@ -134,26 +131,26 @@ With cozy-client, you don't have to deal with the index creation: it is expresse
 const queryDef = client
   .find("io.cozy.todos")
```
Comment (paultranvan) [inline src/tutorials/data/queries.md:132]: gotcha!

File: src/tutorials/data/queries.md [hunk 8]
```diff
@@ -204,15 +201,16 @@ With cozy-client, a sort is expressed through the `sortBy` method. It takes as a
...
```
Comment (paultranvan) [inline src/tutorials/data/queries.md:202]: gotcha!

**Review by paultranvan — COMMENTED** [review]
Still some client.find ;)

**Comment by JulienMirval** [comment]
> Still some client.find ;)

Travis wasn't done I guess, or the fact that I squashed the new commit with the previous one

**Comment by Crash--** [comment]
ping @JulienMirval @paultranvan

**Pull request merged**